cli/test/README.md

```diff
@@ -27,21 +27,65 @@ This document includes CLI commands and the expected CLI output in order to help
+|`fastddsqosprof file.xml set datawriter[profile]`|`ERROR: datawriter must not be FINAL element` [^1]|`DATAWRITER_USAGE`|
+|`fastddsqosprof file.xml set datawriter["a profile"]`|`ERROR: datawriter must not be FINAL element` [^1]|`DATAWRITER_USAGE`|
+|`fastddsqosprof file.xml set datawriter[profile].allocations`|**PENDING** (currently `DataWriter allocations configuration not yet supported`)|N/A|
+|`fastddsqosprof file.xml set datawriter[profile].binary_property`|**PENDING** (currently `DataWriter binary properties configuration not yet supported`)|N/A|
+|`fastddsqosprof file.xml set datawriter[profile].default_profile`|**PENDING** (currently `DataWriter default attribute configuration not yet supported`)|N/A|
+|`fastddsqosprof file.xml set datawriter[profile].entity_id`|**PENDING** (currently `DataWriter entity ID configuration not yet supported`)|N/A|
+|`fastddsqosprof file.xml set datawriter[profile].expect_inline_qos`|`ERROR: expect_inline_qos element not recognized`|`DATAWRITER_USAGE`|
+|`fastddsqosprof file.xml set datawriter[profile].external_locators`|**PENDING** (currently `DataWriter default attribute configuration not yet supported`)|N/A|
+|`fastddsqosprof file.xml set datawriter[profile].history_memory_policy`|**PENDING** (currently `DataWriter history memory policy configuration not yet supported`)|N/A|
+|`fastddsqosprof file.xml set datawriter[profile].ignore_non_matching_locators`|**PENDING** (currently `DataWriter ignore non matching locators configuration flag not yet supported`)|N/A|
+|`fastddsqosprof file.xml set datawriter[profile].locators`|**PENDING** (currently `DataWriter locators configuration not yet supported`)|N/A|
+|`fastddsqosprof file.xml set datawriter[profile].property`|**PENDING** (currently `DataWriter properties configuration not yet supported`)|N/A|
+|`fastddsqosprof file.xml set datawriter[profile].qos`|**PENDING** (currently `DataWriter QoS configuration not yet supported`)|N/A|
+|`fastddsqosprof file.xml set datawriter[profile].times`|**PENDING** (currently `DataWriter time related parameters configuration not yet supported`)|N/A|
+|`fastddsqosprof file.xml set datawriter[profile].user_defined_id`|**PENDING** (currently `DataWriter user defined ID configuration not yet supported`)|N/A|
+|`fastddsqosprof file.xml set datawriter[profile].other`|`ERROR: other element not recognized`|`DATAWRITER_USAGE`|
+|`fastddsqosprof file.xml set datawriter[profile].other --help`|N/A|`DATAWRITER_USAGE`|
 |`fastddsqosprof file.xml set participant -h`|N/A|`PARTICIPANT_USAGE`|
 |`fastddsqosprof file.xml set participant --help`|N/A|`PARTICIPANT_USAGE`|
 |`fastddsqosprof file.xml set participant help`|N/A|`PARTICIPANT_USAGE`|
 |`fastddsqosprof file.xml set participant argument1 argument2 help`|N/A|`PARTICIPANT_USAGE`|
 |`fastddsqosprof file.xml set participant.name`|`ERROR: participant requires a profile name`|`PARTICIPANT_USAGE`|
```

Comment:
```suggestion
|`fastddsqosprof file.xml set participant.name`|`ERROR: participant requires a profile name` [^1]|`PARTICIPANT_USAGE`|
```

Reply:
Setting a participant name requires a profile name. `default_profile` attribute is the one affected by footnote `[^1]`.